### ISSUE TITLE
Protection rune useless

### ISSUE THREAD

**Issue by Lanhild** [obl3pplifp/BlS-issue-tracker#272]
After few tests with ScoutTrooper:
We came to the conclusion that when a mob is protected with a rune and another player hits it with an expert coreglass pickaxe, the mob **disappears**.  (Works with all mobs that can be protected)

**Comment by TankTrooper** [comment]
I believe that this has something to do with the animals w/ lower health not actually taking damage, but instead being instantly killed. The protection rune (I assume) checks for damage, not death. But i'd have to look into that

**Comment by Lanhild** [comment]
Nope, probably not that, as I tried to kill an elephant which has rather high HP, and managed to kill him after 5-6 expert coreglass pickaxe hits.

**Comment by TankTrooper** [comment]
Oh, well then hell if I know what's wrong xD

**Comment by akozlovskiy119** [comment]
This was fixed in https://notabug.org/TenPlus1/mobs_redo/commit/67e4ede31b9450dee2f65827493255199110e062.

BLS `mobs_redo` fork is not in sync with upstream and should be updated.
